libnetdata/socket/socket.c

```diff
@@ -1051,9 +1089,10 @@ int accept_socket(int fd, int flags, char *client_ip, size_t ipsize, char *clien
                 client_ip[ipsize - 1] = '\0';
             }
 
-            if(unlikely(!simple_pattern_matches(access_list, client_ip))) {
+            // Allow patterns to match against either the resolved hostname or the numeric ip address.
+            if(unlikely(!simple_pattern_matches(access_list, client_ip) && (client_host[0]==0 || !simple_pattern_matches(access_list, client_host)))) {
```

Comment:
For a publicly accessible (port-scannable) host, I'm not sure the `unlikely()` macro is of much value here. I guess it really depends on the host etc.

Reply:
I agree - saving the branch misprediction cost is insignificant compared to logging the error.